docs-src/docs/troubleshooting/commonissues.md

```diff
@@ -1,5 +1,73 @@
 #Common issues and their solutions
 
+##Manually deleting a Layer0 instance from AWS
+
+Sometimes, your Layer0 instance might get into an unresponsive and unrecoverable state. This section describes the AWS resources that are created when you create a Layer0 instance and what resources needs to be removed by hand in such an event.
```

Comment:
At the risk of impact the conciseness of this, having a better definition around 'unresponsive' and 'unrecoverable' would be wise. For example, there are some cases where simply re-creating the API is enough to get things working again, in which case these manual steps wouldn't be needed.

Reply:
I could add some tips on commands to run to refresh the instance, like running `l0-setup plan`/`l0-setup apply` to rebuild the instance.

Reply:
:white_check_mark: